lib/routes/app_pages.dart

```diff
@@ -44,6 +45,10 @@ abstract class AppPages {
     GetPage(
       name: Routes.PIN_ENTRY,
       page: () => PinEntry(),
-    )
+    ),
+    GetPage(
+      name: Routes.AVAILABLE_FSP,
```

Comment:
I think it would be clearer to name this something like account linking entry

Reply:
Agreed